website/templates/public/pages/meeting_body.mako

```diff
@@ -1,4 +1,6 @@
-<h2 style="padding-bottom: 30px;">${ meeting['name'] } Posters & Talks</h2>
+<h2 style="padding-bottom: 30px;">${ meeting['name'] }
+    <a>Posters & Talks if meeting['poster'] and meeting['talk'] else Posters if meeting['poster'] else Talks</a>
```

Comment:
This doesn't appear correct; there are no `${}` delimiters.

Please make sure to test this locally.